cluster/v1alpha1/types.go

```diff
@@ -267,6 +271,24 @@ type ClusterClaimSelector struct {
 	MatchExpressions []metav1.LabelSelectorRequirement `json:"matchExpressions,omitempty"`
 }
 
+// PrioritizerConfig represents the configuration of prioritizer
+type PrioritizerConfig struct {
+	// Name is the name of a prioritizer.
+	// The valid names are: balance, steady, ResourceRatioCPU, ResourceAllocatableCPU, ResourceRatioMemory, ResourceAllocatableMemory.
```

Comment:
s/balance/Balance/
s/steady/Steady

I would think we also add some explanations on each.

Reply:
@qiujian16 thanks for the comment, update in the latest code.